README.md

```diff
@@ -29,7 +29,7 @@ Documentation site for Terra. Terra is the core UI component library, unifying U
 
 ### Install Application
 ```
-git clone https://github.cerner.com/terra/terra-ui.git
+git clone https://github.com/terra/terra-ui.git
```

Comment:
Thanks for the contribution @bmitchinson. This should be `https://github.com/cerner/terra-ui.git`, need to replace `github.com/terra` with `github.com/cerner`.